integration-tests/test/shared/stress-test-helpers.ts

```diff
@@ -23,7 +23,7 @@ export const fundRandomWallet = async (
   const fundTx = await env.l1Wallet.sendTransaction({
     gasLimit: 25_000,
     to: wallet.address,
-    gasPrice: await gasPriceForL1(env),
+    gasPrice: await gasPriceForL1(),
```

Comment:
Gas estimation on L1 is failing? That is pretty strange to be honest

Reply:
This just lets us specify a gas price manually if we want.